core/src/test/java/com/exadel/aem/toolkit/core/injectors/RequestPropertyInjectorTestBase.java

```diff
@@ -63,7 +63,9 @@ abstract class RequestPropertyInjectorTestBase {
     private static final String MODELS_PACKAGE_NAME = CoreConstants.ROOT_PACKAGE + ".core.injectors.models.requestproperty";
 
     private static final String EXPECTED_STRING = "Hello World";
+    private static final String EXPECTED_DEFAULT_STRING = "default";
```

Comment:
Matter of taste, but I would resuse the same "expected" strings, arrays, numbers and so on for defaults. 
There is not much sense in creating toitally different constants. 
"Default" fields in test classes are different fields. So, there's no need to compare a "default" value to a "non-default" value in frames of a single field. So, we don't need to differently name values for "default" fields. And we can have less boilerplate constants

Reply:
I think better to have different values, the same values can a little confuse